lib/common/actions/index.js

```diff
@@ -131,14 +132,56 @@ function getErrorMessageFromJson (
     }
     // re-assign error message after it gets used in the detail.
     errorMessage = json.message || JSON.stringify(json)
+    if (json.detail && json.detail.includes('Referenced pattern IDs:')) {
+      // Custom formatting to link to pattern stops to remove if so desired.
+      // Pull the string of patternIds
+      const patternSplit = json.detail.split(/\[(.*?)\]/) // Match text within square brackets (our comma separated list of patternIds)
+      detail = patternSplit[0]
+      const patternsMatch = patternSplit[1]
+      if (patternsMatch) {
+        const patterns = patternsMatch.split(',')
+        // $FlowFixMe
+        detail = <PatternLinkErrorMessage detail={detail} patterns={patterns} />
+      }
+    }
     if (json.detail && json.detail.includes('conflicts with an existing trip id')) {
       action = 'DEFAULT'
     }
   }
-  console.error(errorMessage)
   return { action, detail, message: errorMessage }
 }
 
+const PatternLinkErrorMessage = (props) => {
```

Comment:
Again, I would add a comment here saying that this entire block needs to be removed when the better solution is put in place. I would reference this PR as well so we remember what we were thinking

Reply:
Comment added. Having a component in here itself kind of seems like a sign of the end times.